PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,18 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
```

Comment:
It might be helpful to include these comments in case folks aren't aware that our automated tests don't include cross-browser testing, and that this should be done manually, especially external contributors.

Reply:
:thinking: It seems like that kind of documentation shouldn't be in a PR template.